gateway/exporters/statsd/statsd.go

```diff
@@ -23,7 +23,7 @@ const Name = "statsd"
 var _ exporters.Exporter = new(StatsdExporter)
 
 type Metric struct {
-	Account     string            `json:"Account"`
+	Account     string            `json:",omitempty"`
```

Comment:
I guess you could just remove the json annotation if we aren't planning to include this field in the json.

Reply:
it's required for the metric json but not for the event log json as well, so this is why it's only being set for metrics